helm/cert-manager/values.yaml

```diff
@@ -608,7 +608,16 @@ cainjector:
 
   verticalPodAutoscaler:
     enabled: true
-
+    controlledValues: "RequestsAndLimits"
+    mode: "Auto"
+    updatePolicy:
+      updateMode: "Auto"
+    minAllowed:
+      cpu: ""
+      memory: ""
+    maxAllowed:
+      cpu: ""
+      memory: ""
```

Comment:
When this is set to empty, does it mean unlimited?

Reply:
we may have to omit this field entirely